content/departments/product-engineering/strategy-goals/index.md

```diff
@@ -34,6 +34,7 @@ Here's how things will look a year from now:
 - **Data-driven decisions:** We must understand how users are using and failing to use our product in each use case. To do that, we must improve the maturity of our product user analytics system, aligning this with a well-defined customer journey, while respecting the privacy rights and expectations of our users.
 - **Share underlying capabilities:** We will build solutions that enable a 10x improvement in these areas by building on our code search, navigation, and large-scale changesets capabilities, and our ability to integrate context from best-in-class third-party tools through our open extension API. Users should be able to leverage these capabilities consistently wherever they choose to use Sourcegraph: in the webapp, on their code host, in their editor, on the command line, or through the API.
 - **Low-friction adoption:** We must bring to maturity our two delivery modes of self-hosted and cloud to minimize adoption friction and ensure we meet customers' reliability, security, and compliance needs. No product, however great the feature set, is useful if a customer is unable to use it for any of these reasons.
+- **Eating our own dogfood:** We are developers building developer tools. Nobody is more available to provide immediate, honest, and direct feedback to improve our product than ourselves. [Our vision](#vision) on how well we tackle each of the 5 use cases for our customers certainly applies to us internally at Sourcegraph.
```

Comment:
all-else equal, I would like to avoid using the `dogfood` idiom and try to phrase this in plain English (partly because it is not really an appealing metaphor, and partly because I think it is good to be as descriptive as possible for an international team/audience.

```suggestion
- **Using our own product and providing feedback:** We are developers building developer tools. Nobody is more available to provide immediate, honest, and direct feedback to improve our product than ourselves. [Our vision](#vision) on how well we improve each of the 5 use cases for our customers certainly applies to us internally at Sourcegraph.
```

Reply:
I personally agree that it's not a very appealing metaphor. That said, this is a phrase that is very commonly used in our industry though

Reply:
Maybe say "dogfooding our own product" instead of "eating our own dogfood" and then linking to an external article with more information on what "dogfooding" is could work?

Reply:
What is your concern with not using the word `dogfood` in what we document here?

Reply:
Main reason I would want to reference the term dogfooding is that I believe it's a succinct term that is widely accepted in industry. Perhaps an extreme example, but for me it's kind of like why we use the term Cloud instead of "on demand availability of computer system resources"

I can change it if you feel strongly about not using the term dogfooding (but my preference is mention dogfooding)

Reply:
I incorporated your wording and added dogfooding in parenthesis